packages/genesys-spark-components/src/components/stable/gux-icon/icon-names-deprecated.ts

```diff
@@ -171,27 +171,41 @@ export const deprecatedIconNamesMap: { [key: string]: string } = {
   'external-link': 'fa/arrow-up-right-from-square-regular',
   'fa/00-solid': 'fa/00-regular',
   'fa/arrows-rotate-solid': 'fa/arrows-rotate-regular',
-  'fa/ban-regular': 'deprecated/fa/ban-regular',
+  'fa/asterix-regular': 'fa/asterisk-regular',
+  'fa/ban-outline': 'fa/ban-regular',
   'fa/check-solid': 'custom/check-small-regular',
   'fa/circle-check-regular': 'deprecated/fa/circle-check-regular',
   'fa/circle-xmark-regular': 'deprecated/fa/circle-xmark-regular',
   'fa/clock-outline': 'fa/clock-regular',
   'fa/comment-dots-regular': 'fa/message-dots-regular',
   'fa/comments-regular': 'fa/messages-regular',
-  'fa/diagram-project-gear-regular': 'fa/regular-diagram-project-gear',
+  'fa/diagram-project-gear-regular':
+    'deprecated/fa/diagram-project-gear-regular',
   'fa/eye-regular': 'deprecated/fa/eye-regular',
   'fa/eye-slash-regular': 'deprecated/fa/eye-slash-regular',
+  'fa/home-regular': 'fa/house-regular',
   'fa/life-ring-regular': 'deprecated/fa/life-ring-regular',
   'fa/link-horizontal-slash-regular': 'fa/link-slash-regular',
   'fa/link-horizontal-solid': 'fa/link-regular',
   'fa/list-radio-clean-regular': 'fa/list-ul-regular',
   'fa/list-radio-regular': 'fa/list-ul-regular',
   'fa/list-regular': 'fa/list-ul-regular',
   'fa/rectangle-list-regular': 'fa/square-list-regular',
+  'fa/regular-diagram-project-gear': 'custom/diagram-project-gear-regular',
```

Comment:
This and deprecated/fa/diagram-project-gear-regular are identical. Do we need both?